e2e_test/run_all_tests.sh

```diff
@@ -2,19 +2,20 @@
 set -eo pipefail
 
 SCRIPT_DIR=$(readlink -f "$(dirname "$0")")
+source "$SCRIPT_DIR/shared.sh"
+source "$SCRIPT_DIR/debug-fee-currency/lib.sh"
+
 TEST_GLOB=$1
 
 ## Start geth
 cd "$SCRIPT_DIR/.." || exit 1
 make geth
-trap 'kill %%' EXIT  # kill bg job at exit
-build/bin/geth --dev --http --http.api eth,web3,net &> "$SCRIPT_DIR/geth.log" &
+trap 'kill %%' EXIT # kill bg job at exit
+build/bin/geth --dev --dev.period 1 --http --http.api eth,web3,net &>"$SCRIPT_DIR/geth.log" &
```

Comment:
Couldn't we trigger block building by sending dummy requests where needed instead of enabling this for all tests?